networkx/utils/backends.py

```
@@ -600,7 +603,14 @@ def wrapper(*args, **kwds):
                     graph_name=gname,
                 )
 
-        result = getattr(backend, name).__call__(*bound.args, **bound.kwargs)
+        try:
+            result = getattr(backend, name).__call__(*bound.args, **bound.kwargs)
+        except NotImplementedError as exc:
+            if fallback_to_nx:
+                return func(*args, **kwds)
+            pytest.xfail(
+                exc.args[0] if exc.args else f"{name} raised NotImplementedError"
+            )
```

We could also raise the original exception here. I don't have a preference.